src/main/java/com/thealgorithms/sorts/RadixSort.java

```diff
@@ -1,62 +1,97 @@
 package com.thealgorithms.sorts;
 
+import com.thealgorithms.maths.NumberOfDigits;
 import java.util.Arrays;
 
-final class RadixSort {
+/**
+ * This class provides an implementation of the radix sort algorithm.
+ * It sorts an array of positive integers in increasing order.
+ */
+public final class RadixSort {
+    private static final int BASE = 10;
+
     private RadixSort() {
     }
 
-    private static int getMax(int[] arr, int n) {
-        int mx = arr[0];
-        for (int i = 1; i < n; i++) {
-            if (arr[i] > mx) {
-                mx = arr[i];
-            }
+    /**
+     * Sorts an array of positive integers using the radix sort algorithm.
```

Comment:
```suggestion
     * Sorts an array of nonnegative integers using the radix sort algorithm.
```

Reply:
Thanks, this is a more accurate description.